src/ProgressOnderwijsUtils/SchemaReflection/DatabaseDescription.cs

```diff
@@ -32,17 +32,18 @@ public static DbNamedObjectId[] LoadAllObjectsOfType(SqlConnection conn, string
 
     public sealed class DatabaseDescription
     {
-
         readonly IReadOnlyDictionary<DbObjectId, Table> tableById;
         readonly IReadOnlyDictionary<DbObjectId, View> viewById;
+        readonly IReadOnlyDictionary<DbObjectId, CheckConstraint> checkConstraintById;

```

Comment:
Doe je ooit iets met de checkConstraintId?  Kun je daar op zoeken met deze api?

Reply:
Hij wordt gebruikt bij 
```
public IEnumerable<CheckConstraint> AllCheckConstraints
            => checkConstraintById.Values;
```

Identiek aan AllViews en viewById. Ik verwacht wel om AllAconstraints straks in de EnsureConstraints in PNet te gebruiken

Reply:
Dus je kunt er niet op zoeken via id, toch?

Reply:
Gezien de O(n), wil je een lookup gebruiken.

Reply:
klopt